### ISSUE TITLE
Too much logo

### ISSUE THREAD

**Issue by adamgall** [decentdao/decent-interface#1879]
@decentdao/design @decentdao/product

**Homepage UI feedback**

I love seeing our logo more than just about anyone, but I don't think its clean to have three instances of our logo throughout the home screen of the app (currently we have our logomark in the top left corner, logo above the home page UI and text field, and "Made with love..." in the footer).

My suggestion would be to remove the "decent" logo in the UI (the centered one above the Safe finder field on the home page).

Combining this with my comment re: the Safe finder field UI above. Perhaps we just remove the decent logo and the Safe finder field from this homepage (see second screenshot) to have the cleanest experience on app load.

<img width="1710" alt="Screenshot 2024-05-22 at 09 24 45" src="https://github.com/decentdao/decent-interface/assets/13035147/af2aa796-156f-4bca-a419-f28d9c636448">

We could remove these components:
<img width="1298" alt="Screenshot 2024-05-22 at 09 27 51" src="https://github.com/decentdao/decent-interface/assets/13035147/228662a4-cdc2-4d14-ae92-b59c20958b59">

So the core homepage UI could just be this:
<img width="1298" alt="Screenshot 2024-05-22 at 09 30 24" src="https://github.com/decentdao/decent-interface/assets/13035147/0b5377d8-bc6b-4edd-a3b0-d2ffe5975fac">

Perhaps we could also experiment then with replacing the logomark in the top left corner with a complete logo (smaller height, the top nav is a bit fat).

TLDR;
- Yank "decent" logo in homepage UI
- Yank second Safe finder form field
- Replace "d" in top left with a shorter (by height) "decent" logo

_Originally posted by @parkermccurley in https://github.com/decentdao/decent-interface/discussions/1863#discussioncomment-9524457_

See original comment thread

**Comment by adamgall** [comment]
This is on @decentdao/design to take first pass.

**Comment by nicolaus-sherrill** [comment]
![Image](https://github.com/decentdao/decent-interface/assets/62394931/2e442286-3dfd-4f76-a4d7-b36620f2e6ad)
![Image](https://github.com/decentdao/decent-interface/assets/62394931/db80a845-bd77-40a8-95fc-239f34278dd3)

**Comment by xraystyle1980** [comment]
Closing this issue to open a new issue that address multiple home design updates (design update included for reference)
![image](https://github.com/decentdao/decent-interface/assets/18615633/223e33f6-3ee2-414d-8a96-b49e716e8b8b)
